tests/test_generators/input/dynamic-enums-example.yaml

```diff
@@ -0,0 +1,171 @@
+id: https://w3id.org/linkml/examples/enums
```

Comment:
@cmungall Thanks for reminding me where this file is. I agree this is better to use. I copied it over. It's just a test input file, so that's probably alright, but if you'd like a DRYer approach where I import this from `linkml-model` somehow, let me know.